### DISCUSSION TITLE
Tryton Integration

### DISCUSSION THREAD

**Discussion by rboixader10** [getsentry/sentry-python#1821] [Q&A]
Hello, can someone help me on how to do a Tryton integration?
I have installed self-hosted sentry
Then I've created the wsgi.py file and run it:
![wsgi](https://user-images.githubusercontent.com/106231540/210511161-ae75cffa-af45-47ae-b853-fe7a767d0775.png)
But I can't show in sentry neither the tryton logs nor the errors.

**Comment by antonpirker** [comment]
Hey @rboixader10 

The code looks OK. 
Can you add an error just below the call to `sentry_sdk.init()` to see if this "normal" error is showing up in Sentry? Something like:

```python
my_error = 1 / 0
```

**Comment by antonpirker** [comment]
Maybe also adding `debug=True` to your call to `sentry_sdk.init()` will output some useful information when your process starts.
